tests/zenko_e2e/backbeat/tests/crr-pause-resume/awsBackend.js

```diff
@@ -44,6 +44,7 @@ describe('Replication Pause-Resume with AWS backend', function() {
         next => scalityUtils.waitUntilReplicated(srcBucket, key,
             undefined, next),
         next => backbeatAPIUtils.pauseReplication(null, next),
+        next => setTimeout(next, 5000),
```

Comment:
I think this one should be removed, as we expect the status to be `disabled` right away.

Reply:
after discussion, let's keep them (since publish/subscribe of redis may introduce delay)